build/lib/builder.js

```diff
@@ -19,7 +19,7 @@ const TMP_DIR = path.join(DIST_DIR, 'tmp');
 // Platforms/OS mappings
 // NOTE: 'linux' could be added, but is not officially supported.
 // Specifying --all will only produce a linux output on a linux host.
-const ALL_OSES = Array.from(new Set([ 'win32', 'osx', thisOS() ]));
+const ALL_OSES = [ 'win32', 'osx', 'linux' ];
```

Comment:
May want to remove the _NOTE_ above?

I'm also wondering if `--all` should explicitly include a platform we don't support? If we've fixed our build page, maybe this should stay as is?

Reply:
we haven't fixed our build page, or modified our release scripts to expect this (no linux zip) AFAIK. So not having this will break both still. (And may break some CI builds where I think we may build say an Android native module without unit tests so can use a linux node/sdk)

Reply:
appcelerator/titanium_build/pull/3 fixes the build page, I've just pushed a commit that removes that comment too